src/main/java/nl/ordina/jobcrawler/controller/JobcrawlerController.java

```diff
@@ -53,6 +58,19 @@ public Vacancy getByID(@PathVariable("id") UUID id) {
         return vacancy.orElseThrow(() -> new VacancyNotFoundException("Vacancy with id: " + id + " not found."));
     }
 
+    @GetMapping("/scrape")
```

Comment:
Als dit een aanroep is die data aan de backend verandert (en dat lijkt er wel op), dan is een GET niet de juiste manier om deze aan te roepen. 
Zie ook https://www.w3schools.com/tags/ref_httpmethods.asp en https://restfulapi.net/idempotent-rest-apis/